components/shared/link-component.tsx

```diff
@@ -1,35 +1,35 @@
 'use client'
-import React, { ReactNode } from 'react'
 
-import classNames from 'clsx'
+import { HTMLAttributes } from 'react'
+
 import Link from 'next/link'
```

Comment:
To prevent having to name it LinkComponent you could do.

`import { default as NextLink } from "next/link";`